lib/cc/engine/analyzers/javascript/main.rb

```diff
@@ -16,7 +16,7 @@ class Main < CC::Engine::Analyzers::Base
           ]
           LANGUAGE = "javascript"
           DEFAULT_MASS_THRESHOLD = 40
-          BASE_POINTS = 3000
+          POINTS_PER_OVERAGE = 30_000
```

Comment:
The points per overage vary by language (roughly a function of mass_threshold / granularity of node parsing).

Since the BASE_POINTS and the formula are the same, they can be refactored to live in the Base Class.